Sources/Commands/Describe.swift

```diff
@@ -27,7 +27,7 @@ struct DescribedPackage: Encodable {
     let swiftLanguagesVersions: [String]?
 
     init(from package: Package) {
-        self.name = package.manifestName // TODO: rename property to manifestName?
+        self.name = package.manifest.displayName // TODO: rename property to manifestName?
```

Comment:
Renaming to `manifestName` seems good to me, but I guess that would be an API change (since it shows up in the JSON).  Maybe we can include both for a while?  I don't think tools version is necessary correct here, because the API is on the output, not the package.  Perhaps an option to the describe command?